Documentation/devel/stage1-implementors-guide.md

```diff
@@ -104,12 +104,25 @@ For example, it removes the network namespace of a pod.
 * `--debug` to activate debugging
 * UUID of the pod
 
+### `rkt stop` => "coreos.com/rkt/stage1/stop"
+
+The optional stop entrypoint initiates an orderly shutdown of stage1.
+
+In the bundled rkt stage 1, the entrypoint is calling `systemctl halt` on the container (through SSH).
+
+This entrypoint was added in interface version 3.
```

Comment:
oh, I didn't see the call to `interfaceVersionSupportsStop()`. But still, I am not sure it is necessary for adding a new optional entrypoint.